terraform/modules/alertmanager/alertmanager-service.tf

```diff
@@ -112,6 +112,39 @@ resource "aws_ecs_service" "alertmanager_nlb" {
   }
 }
 
+resource "aws_ecs_service" "alertmanager_alb" {
+  for_each = {
+    for _, subnet in data.aws_subnet.private_subnets :
+    subnet.id => subnet.availability_zone
+  }
+  name            = "${var.environment}-alertmanager-alb-${each.value}"
+  cluster         = "${var.environment}-ecs-monitoring"
+  task_definition = aws_ecs_task_definition.alertmanager_nlb.arn
+  desired_count   = 1
+  launch_type     = "FARGATE"
+
+  load_balancer {
+    target_group_arn = aws_lb_target_group.alertmanager.arn
+    container_name   = "alertmanager"
+    container_port   = 9093
+  }
+
+  load_balancer {
+    target_group_arn = aws_lb_target_group.alertmanager_per_az[each.value].arn
```

Comment:
am I being blind or does this target group not exist?

Reply:
it's in `alb.tf`

Reply:
...which is an untracked file. D'oh!

Reply:
I can confirm `alertmanager_per_az` does not exist.

Reply:
now added